tests/conftest.py

```diff
@@ -44,3 +46,14 @@ def client_sandbox_enabled():
     Client instance loads PDL_API_KEY from .env file and Sandbox enabled.
     """
     return PDLPY(sandbox=True)
+
+
+@pytest.fixture
+def client_env_test():
+    """
+    Client instance loads correct env variables.
+    """
+    os.environ["VERSION"] = "PDL_TEST_FAIL"
+    os.environ["PDL_VERSION"] = "v6"
```

Comment:
will this update to os.environ leak outside this test?

Reply:
Good point. Updated the code to unset the variables.

Reply:
if the environment had a value for VERSION before this test was executed, this will clear it I think, which is kind of not cool